### ISSUE TITLE
새로 수정된 Kspace data에 오류가 있습니다 (read시 오류 발생)

### ISSUE THREAD

**Issue by jellyheadandrew** [LISTatSNU/FastMRI_challenge#183]
![image](https://github.com/LISTatSNU/FastMRI_challenge/assets/80747187/8dedc9cf-ad29-42f6-bf6e-9662cda162a2)

안녕하세요, iabeng34 node를 사용하고 있는 팀입니다.

2023/07/11에 재업로드된 kspace file중 하나가 읽었을 때 위와 같이 에러가 발생합니다.

재배포 부탁드립니다.

감사합니다.

**Comment by wogur110** [comment]
안녕하세요, 2023 SNU FastMRI Challenge 조교 배재혁입니다.

말씀해주신 read 오류가 "/Data/train/kspace/brain_acc4_140.h5" 에서만 발생하는지, 다른 파일에서도 발생하는지 확인해주시면 감사하겠습니다.

감사합니다.
배재혁 드림

**Comment by jellyheadandrew** [comment]
이 issue를 올린 이후 아무 조작도 하지 않았습니다. data loading중의 문제인 것 같은데, 가능하시다면 빠르게 해결 부탁드립니다.

감사합니다.

**Comment by jellyheadandrew** [comment]
iabeng34 node 문제 해결되었나요? workspace에서 jupyter을 다시 열려고 해도 아직 문제가 발생하여 문의드립니다.

**Comment by wogur110** [comment]
해당 문제와 관련해서 iaebeng34 노드 참가자에게 메일을 보냈습니다. 다른 팀원 분의 계정을 Vessl에 초대하였으니 새로운 노드를 만들어서 우선 진행해 주시면 감사하겠습니다.

**Comment by jellyheadandrew** [comment]
그렇게 하면 코드는 모두 사라지지 않나요?

**Comment by wogur110** [comment]
네 당장은 jupyter 로 접속할 수 있는 방법이 없습니다. 우선 새로우 노드로 새롭게 코드를 돌리시고 iabeng34 노드가 복구되는 대로 다시 활용하시는 것이 가장 빠른 방법으로 생각됩니다. 불편을 드려 죄송합니다.

감사합니다.
배재혁 드림

**Comment by Giventicket** [comment]
안녕하세요, 2023 SNU FastMRI Challenge 조교 서준표입니다.

iabeng34 node 문제 해결되었습니다.
불편을 드려 죄송합니다.

좋은 결과있길 응원합니다.

감사합니다.
서준표 드림.
